Add `osu! UK Tournament` wiki articles (#5692)
ppy/osu-wiki: wiki/Tournaments/OUKT/3/en.md

| **Gadrin** ![][flag_GB] | **3** | 1 | ![][flag_GB] -MoeYandere- | |
| Chungygiri ![][flag_GB] | 0 | **1** | ![][flag_GB] **DatPenguinTho** | Forfeit |
| **EatYourMattress** ![][flag_GB] | **3** | 1 | ![][flag_GB] Crai | |
| kozuka ![][flag_GB] | 1 | **3** | ![][flag_GB] **Limit_Break** | |
| **kyou** ![][flag_GB] | **3** | 0 | ![][flag_GB] dreeeaam | |
| **ChicagoTed7172** ![][flag_GB] | **3** | 0 | ![][flag_GB] cooki3zz | |
| Scootawoo ![][flag_GB] | 2 | **3** | ![][flag_GB] **iLikeMudkipz** | |

## Ruleset

### General Tournament Rules

- Players that are under rank #50,000 are not allowed to sign up.
- UK staff are allowed to play.
- This tournament is a single elimination 1v1 tournament. Lose once, you are out. Brackets will be determined by the Draw Master, see Seeding Rules.
- The tournaments shall have a map pool for each phase of the tournament. The next map pool in the tournament shall be announced following the end of the preceding round.
- Each map pool will have 5-9 no mod picks, 2 mod picks, a freemod bracket and 1 tiebreaker The mod brackets shall be Hidden/Double Time/Hard Rock/Freemod. Two maps for each.

> - Each player may only pick 1 map from each mod pool to try and encourage variety in the match. For example: P1 picks a HD map, he cant pick from the HD pool again. P2 may however pick a HD map.

Reviewer: ```suggestion
- Each player may only pick 1 map from each mod pool to try and encourage variety in the match. For example: P1 picks a HD map, he can't pick from the HD pool again. P2 may however pick a HD map.
```

- If a player disconnects from the game, a regame will be offered. Ragequitters/quits will forfeit that map. This is relating to genuine internet disconnects/cuts/failures.
- RO64/RO32 will be best of 5. RO16 will be best of 7. Quarterfinals/Semifinals will be best of 9. Finals/3rd Place will be best of 11.
- Any unexpected happenings or events not covered by the rules shall be handled by the organizer at their discretion.

### Seeding Rules

Seeding for this tournament will be as follows. This is a tournament of 64 players, 16 of which will be seeded. The draw is then completed as follows:

1. The 1st seed will be placed at the top of the top half of the draw.
2. The 2nd seed will be placed at the bottom of the bottom half of the draw.
3. The 3rd and 4th seeds will be drawn to either the bottom of the top half or the top of the bottom half.
4. The 5th-8th seeds will be drawn in remaining gaps so they do not run into another seed in the first 3 rounds.
5. The 9th-16th seeds will be drawn in remaining gaps so they do not run into another seed in the first 2 rounds.
6. The remaining 48 competitors will be randomly assigned to the remaining gaps in the draw.

### Match Rules

> - The referee shall make the game and give it a appropriate title. First pick shall be decided by !roll, highest roll wins. The pick order then will alternate back and forth between both players until one player wins.

Reviewer: ```suggestion
- The referee shall make the game and give it a appropriate title. First pick shall be decided by `!roll`, the highest roll wins. The pick order then will alternate back and forth between both players until one player wins.
```

- The game mode will be set to Team Vs.
- The referee is not to participate in the actual playing of the beatmaps.
- Players PM the referee with the beatmap they are picking from the map pool.
- After the end of the match, the referee will post the title of the match and a link to the MP history of that lobby and the final match score.
- Matches will be locked and invite only.
- If you fail the map, it is considered as a loss.

### Scheduling Rules/Info

- Matches will be played on the weekend.

> - If you cannot play at the time posted for you, contact the organizer for a re arrangement.

Reviewer: ```suggestion
- If you cannot play at the time posted for you, contact the organizer for a rearrangement.
```
